lib/MusicBrainz/Server/Edit/Role/IPI.pm

```diff
@@ -12,6 +23,62 @@ with 'MusicBrainz::Server::Edit::Role::ValueSet' => {
     extract_value => sub { shift->ipi }
 };
 
+after initialize => sub {
+    my ($self, %opts) = @_;
+
+    my $old_ipis = $self->data->{old}{ipi_codes};
+    my $new_ipis = $self->data->{new}{ipi_codes};
```

Comment:
Aren't these potentially `undef`? De-referencing them below will break if so.

Reply:
Maybe? What should be done, ` // []`?

Reply:
Yes, `// []` (otherwise the line below will throw `Can't use an undefined value as an ARRAY reference`).